### ISSUE TITLE
Implement markup parser corner case

### ISSUE THREAD

**Issue by trevorbaca** [Abjad/abjad#397]
The following is correct LilyPond markup:

```
\note #"4" #1
```

(The input generates a quarter note with an up-stem. Note the double quotes)

But this parses incorrectly ...

```
markup = Markup(r'\note #"4" #1')
```

... and gives ...

```
\markup {
    \note
        4
        #1

    }
```

... when formatted.

The markup parser should preserve `#"4"` in preference to `4`.

Note that in this case the LilyPond `note-by-number` command provides a workaround:

```
Markup(r'\note-by-number #2 #0 #1')
```

So the fix is not urgent.

**Comment by josiah-wolf-oberholtzer** [comment]
Looks like the parser is not the problem, but rather how values are formatted:

```
>>> markup = Markup(r'\note #"4" #1')
>>> markup
Markup(contents=(MarkupCommand('note', '4', 1),))
```

Note above that `'4'` is correctly parsed as a string. But, when formatting as LilyPond syntax, it displays the undesired behavior you experienced:

```
>>> print(format(markup))
\markup {
    \note
        4
        #1
    }
```

**Comment by josiah-wolf-oberholtzer** [comment]
A fix isn't simple at the moment. A lot has changed in LilyPond's own parser since we implemented ours. Many of those changes made it more pleasant to work with bare numbers.

Correcting this bug may require initiating the long-awaited LilyPondParser update.

**Comment by trevorbaca** [comment]
Still valid in June 2017:
```
markup = Markup(r'\note #"4" #1')                                               
```
Output:
```
>>> f(markup)
\markup {
    \note
        4
        #1
    }
```

**Comment by trevorbaca** [comment]
New functionality is now available in Abjad 3.1 with the `literal=None` markup initializer keyword.

```
    markup = abjad.Markup(
        r'\markup { \note #"4" #1 }',
        direction=abjad.Up,
        literal=True,
        )

    note = abjad.Note("cs'4")
    abjad.attach(markup, note)

    abjad.f(note)
    cs'4
    ^ \markup { \note #"4" #1 }
```

Interim abjad.Markup.from_literal() constructor is now deprecated.
